templates/main/signup.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <title>Signup</title>
+        <link rel="stylesheet" href="css/main.css">
+        <script src="js/login.js" defer></script>
+    </head>
+    <body>
+    	<div id='container'>
+        <h1>Signup page</h1>
+    	 <form action="" method="POST">	  
+			  	
+			    <label for="uname"><b>Username</b></label>
+			    <input type="text" placeholder="Enter Username" name="uname" required>
+			    <br>
+			    <label for="email"><b>Email</b></label>
+			    <input type="text" placeholder="Enter Email" name="email" required>
+			    <br>
+			    <label for="psw"><b>Password</b></label>
+			    <input type="password" placeholder="Enter Password" name="psw" required>
+			    <br>
```

Comment:
Signing up with typing your password only once requires the user to not make any mistakes at all. This may however still happen even if they are paying extra attention to the process. I suggest adding a "Confirm password" field to solve this problem, if you think it should be handled.